docs/framework/release-notes/2024/07-09-july-security-and-quality-rollup.md

```diff
@@ -0,0 +1,105 @@
+The following table is for earlier Windows and Windows Server versions for Security and Quality Rollup updates.  
+
+| Product version | Security and quality rollup |
+| --- | --- |
+| **Windows Server 2012 R2** | **[5041023](https://support.microsoft.com/kb/5041023)** |
+| .NET Framework 3.5 | [5039910](https://support.microsoft.com/kb/5039910) |
+| .NET Framework 4.6.2, 4.7, 4.7.1, 4.7.2 | [5039881](https://support.microsoft.com/kb/5039881) |
+| .NET Framework 4.8 | [5039890](https://support.microsoft.com/kb/5039890) |
+| **Windows Server 2012** | **[5041022](https://support.microsoft.com/kb/5041022)** |
+| .NET Framework 3.5 | [5039908](https://support.microsoft.com/kb/5039908) |
+| .NET Framework 4.6.2, 4.7, 4.7.1, 4.7.2 | [5039880](https://support.microsoft.com/kb/5039880) |
+| .NET Framework 4.8 | [5039888](https://support.microsoft.com/kb/5039888) |
+| **Windows Server 2008 R2** | **[5041021](https://support.microsoft.com/kb/5041021)** |
+| .NET Framework 3.5.1 | [5039909](https://support.microsoft.com/kb/5039909) |
+| .NET Framework 4.6.2, 4.7, 4.7.1, 4.7.2 | [5039882](https://support.microsoft.com/kb/5039909)|
+| .NET Framework 4.8 |[5039891](https://support.microsoft.com/kb/5039891) |
+| **Windows Server 2008** | **[5041024](https://support.microsoft.com/kb/5041024)** |
+| .NET Framework 2.0, 3.0 | [5039911](https://support.microsoft.com/kb/5039911) |
+| .NET Framework 3.5 SP1 | [5040673](https://support.microsoft.com/kb/5040673) |
+| .NET Framework 4.6.2 | [5039882](https://support.microsoft.com/kb/5039882) |
+
+The following table is for earlier Windows and Windows Server versions for Security Only updates, which aren't cumulative.
+
+| Product version | Security only update |
+| --- | --- |
+| **Windows Server 2008 R2** | **[5041026](https://support.microsoft.com/kb/5041027)** |
+| .NET Framework 3.5.1 | [5040119](https://support.microsoft.com/kb/5040119) |
+| .NET Framework 4.6.2, 4.7, 4.7.1, 4.7.2 | [5040122](https://support.microsoft.com/kb/5040122) |
+| .NET Framework 4.8 |[5040123](https://support.microsoft.com/kb/5040123) |
+| **Windows Server 2008** | **[5041027](https://support.microsoft.com/kb/5041027)** |
+| .NET Framework 2.0, 3.0 | [5040118](https://support.microsoft.com/kb/5040118) |
+| .NET Framework 3.5 SP1 | [5040680](https://support.microsoft.com/kb/5040680) |
+| .NET Framework 4.6.2 | [5040122](https://support.microsoft.com/kb/5040122) |
+
+The operating system row lists a KB which will be used for update offering purposes. When the operating system KB is offered, the applicability logic will determine the specific .NET Framework update(s) will be installed. Updates for individual .NET Framework versions will be installed based on the version of .NET Framework that is already present on the device. Because of this the operating system KB is not expected to be listed as installed updates on the device. The expected update to be installed are the .NET Framework specific version updates listed in the preceding table.
```

Comment:
```suggestion
The operating system rows list a KB that's used for update-offering purposes. When the operating system KB is offered, the applicability logic determines the specific .NET Framework updates that will be installed. Updates for individual .NET Framework versions are installed based on the version of .NET Framework that's already present on the device. Because of this, the operating system KB is not expected to be listed as an installed update on the device. The expected updates to be installed are the .NET Framework&ndash;specific version updates listed in the preceding table.
```